Sphinx theme broken, intersphinx_mapping

To issues related to sphinx:

1. `make html` does not work locally:
```
make html
Running Sphinx v7.0.1
[autosummary] generating autosummary for: applications.rst, auxiliary_files.rst, camera_efficiency.rst, configuration_module.rst, corsika.rst, data_model.rst, databases.rst, db_handler.rst, gammasim-tools-configuration.rst, getting_started.rst, ..., layout_array.rst, library.rst, mc_model.rst, psf_analysis.rst, ray_tracing.rst, sim_telarray.rst, simulation_software.rst, simulators.rst, util.rst, visualization.rst
building [mo]: targets for 0 po files that are out of date
writing output...
building [html]: targets for 24 source files that are out of date
updating environment: [new config] 24 added, 0 changed, 0 removed
reading sources... [100%] visualization
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  4%] applications
Theme error:
An error happened in rendering the page applications.
Reason: UndefinedError("'style' is undefined")
make: *** [html] Error 2
```
Seems to be a known issue (see [here](https://github.com/readthedocs/sphinx_rtd_theme/issues/1465)). Not sure why it works in the github action.

2. running sphinx gives the following warning:
```
WARNING: The pre-Sphinx 1.0 'intersphinx_mapping' format is deprecated and will be removed in Sphinx 8. Update to the current format as described in the documentation. Hint: "intersphinx_mapping = {'<name>': ('https://docs.python.org/', None)}".https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping
```
Also something we probably want to fix.

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -70,7 +70,7 @@ release = version
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
-    # "sphinx.ext.intersphinx",
+    "sphinx.ext.intersphinx",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
     "sphinx.ext.githubpages",

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -139,7 +139,7 @@ pygments_style = "sphinx"
 # a list of builtin themes.
 #
 # html_theme = "sphinx_rtd_theme"
-html_theme = "classic"
+html_theme = "bizstyle"
 
 html_title = f"{project} v{version} Manual"
 
@@ -235,7 +235,7 @@ texinfo_documents = [
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-# intersphinx_mapping = {"https://docs.python.org/": None}
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
 
 
 # -- Raul: making sphinx add classes' __init__ always

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -70,7 +70,7 @@ release = version
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
-    "sphinx.ext.intersphinx",
+    # "sphinx.ext.intersphinx",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
     "sphinx.ext.githubpages",
@@ -138,7 +138,8 @@ pygments_style = "sphinx"
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+# html_theme = "sphinx_rtd_theme"
+html_theme = "classic"
 
 html_title = f"{project} v{version} Manual"
 
@@ -234,7 +235,7 @@ texinfo_documents = [
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"https://docs.python.org/": None}
+# intersphinx_mapping = {"https://docs.python.org/": None}
 
 
 # -- Raul: making sphinx add classes' __init__ always

